website/docs/docs/collaborate/manage-access/audit-log.md

```diff
@@ -5,6 +5,12 @@ description: "You can troubleshoot possible issues and provide security audits b
 sidebar_label: "Audit log"
 ---
 
+:::note
+
+Audit logs are currently not supported for Microsoft Azure.
+
+:::
+
 To review actions performed by people in your organization, dbt provides logs of audited user and system events in real time. The audit log appears as events happen and includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. 
 
```

Comment:
suggest moving the callout from line 8 to here on line 15

Reply:
Good call! Taken care of it